libs/langchain/langchain/retrievers/document_compressors/embeddings_filter.py

```diff
@@ -33,6 +33,8 @@ class EmbeddingsFilter(BaseDocumentCompressor):
     """Threshold for determining when two documents are similar enough
     to be considered redundant. Defaults to None, must be specified if `k` is set
     to None."""
+    return_similarity_scores = False
```

Comment:
probably don't even need to make this configurable, could just always return score in state